docs/sources/alerting/learn/examples/table-data.md

```diff
@@ -0,0 +1,128 @@
+
+{{< admonition type="caution" >}}
+These three conditions must be met—otherwise, Grafana can’t evaluate the table data and the rule will fail.
+{{< /admonition >}}
+
+## Example overview
+
+Imagine you store disk usage in a `DiskSpace` table and you want to trigger alerts when the available space drops below 5%.
+
+| Time       | Host | Disk | PercentFree |
+| ---------- | ---- | ---- | ----------- |
+| 2021-06-07 | web1 | /etc | 3           |
+| 2021-06-07 | web2 | /var | 4           |
+| 2021-06-07 | web3 | /var | 8           |
+
+To calculate the free space per Host and Disk in this case, you can use `$__timeFilter` to filter by time but without returning the date to Grafana:
+
+```sql
+SELECT
+  Host,
+  Disk,
+  AVG(PercentFree) AS PercentFree
+FROM DiskSpace
+WHERE $__timeFilter(Time)
+GROUP BY Host, Disk
+```
+
+This query returns the following table response:
+
+| Host | Disk | PercentFree |
+| ---- | ---- | ----------- |
+| web1 | /etc | 3           |
+| web2 | /var | 4           |
+| web3 | /var | 8           |
+
+When Alerting evaluates the query response, the data is transformed into three alert instances as previously detailed:
+
+- The numeric column becomes the value for the alert condition.
+- Additional columns define the label set for each alert instance.
+
+| Alert instance               | Value |
+| ---------------------------- | ----- |
+| `{Host="web1", Disk="/etc"}` | 3     |
+| `{Host="web2", Disk="/var"}` | 4     |
+| `{Host="web3", Disk="/var"}` | 8     |
+
+Finally, an alert condition that checks for less than 5% of free space (`$A < 5`) would result in two alert instances firing:
+
+| Alert instance               | Value | State  |
+| ---------------------------- | ----- | ------ |
+| `{Host="web1", Disk="/etc"}` | 3     | Firing |
+| `{Host="web2", Disk="/var"}` | 4     | Firing |
+| `{Host="web3", Disk="/var"}` | 8     | Normal |
+
+## Try it with TestData
+
+To test this quickly, you can simulate the table using the [**TestData** data source](ref:testdata-data-source):
+
+1. Add the **TestData** data source through the **Connections** menu.
+1. Go to **Alerting** and create an alert rule
+1. Select **TestData** as the data source.
+1. From **Scenario**, select **CSV Content** and paste this CSV:
+
+   ```bash
+   host, disk, percentFree
+   web1, /etc, 3
+   web2, /var, 4
+   web3, /var, 8
+   ```
+
+1. Set a condition like `$A < 5` and **Preview** the alert.
+
+   Grafana evaluates the table data and fires the two first alert instances.
+
+   {{< figure src="/media/docs/alerting/example-table-data-preview.png" max-width="750px" alt="Alert preview with tabular data using the TestData data source" >}}
+
+## **Differences with time series data**
+
+Working with time series is similar—each series is treated as a separate alert instance, based on its label set.
+
+The key difference is the data format:
+
+- **Time series data** contains multiple values over time, like: `{timestamp, value}, {timestamp, value}, ...`.
```

Comment:
I am not sure the example `{timestamp, value}, {timestamp, value}, ...`  is informative.

Reply:
addressed by https://github.com/grafana/grafana/pull/105325/commits/ca4aec181ec934c4883444bcee87d50b8d35ebe6